src/kani-compiler/rustc_codegen_kani/src/overrides/hooks.rs

```diff
@@ -169,18 +170,24 @@ impl<'tcx> GotocHook<'tcx> for Assert {
         let loc = tcx.codegen_span_option(span);
         let caller_loc = tcx.codegen_caller_span(&span);
 
+        let mut stmts: Vec<Stmt> = Vec::new();
+
+        if tcx.queries.get_check_assertion_reachability() {
+            // inject a reachability (cover) check to the current location
+            let reach_msg = format!("kani_reachability_check for {}", msg);
```

Comment:
Nit: Is this meant to be user visible?

Reply:
No, this is meant to make it easy for the CBMC result parser to match the reachability check with the assertion.

Reply:
Ok. In that case, I would replace the `for` by either `:` or `[]`:
```
let reach_msg = format!("[KANI_REACHABILITY_CHECK] {}", msg);
```
or
```
let reach_msg = format!("KANI_REACHABILITY_CHECK: {}", msg);
```

I think it makes it more clear that this is some sort of tag / category.